spec/helpers/sessions_helper_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe SessionsHelper, type: :helper do
+  let(:user) do
+    User.create(username: 'Bob', email: 'bob@mail.com', password: '123')
+  end
+
+  describe '#log_in' do
+    subject { log_in(user) }
+    let(:logged_in_user) { user.id }
```

Comment:
These `let`s make the test harder to read in my opinion, I would just:
`it { is_expected.to be user.id }`